app/packages/looker-3d/src/fo3d/point-cloud/use-pcd-material.tsx

```
@@ -58,14 +69,20 @@ export const usePcdMaterial = (
       return { min: 0, max: 1 };
     }
 
-    const intensity =
-      geometry.getAttribute("color") ?? geometry.getAttribute("intensity");
-
-    if (intensity) {
-      return computeMinMaxForColorBufferAttribute(intensity);
+    const isLegacyIntensity = !Boolean(geometry.getAttribute("intensity"));
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)